arches/app/views/api.py

```diff
@@ -1434,6 +1434,13 @@ def post(self, request):
         return response
 
 
+class UserIncompleteWorkflows(APIBase):
+    def get(self, request):
+        return JSONResponse({
+            "incomplete_workflows": models.WorkflowHistory.objects.filter(user=request.user, completed=False).exclude(componentdata__iexact='{}')
+        })
+
```

Comment:
Should we order_by() so that we don't have to sort it later on the client?

Reply:
no huge need here, and there's currently no sorting on the client. The ORM returns by order of record creation so things are sorted by the `created` field de-facto. However I've added it in anyway since we're throwing timezones into the mix now.